server/src/utils/validators.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import SurveyDBHandler from '../databaseHandlers/surveyDBHandler';
+import { SurveyResultRequest } from '../models/surveyModel';
+
+export const surveyResultRequestValidator = async (
+  requestData: SurveyResultRequest,
+  surveyDBHandler: SurveyDBHandler
+): Promise<string[]> => {
+  const { callId, acsUserId, response } = requestData;
+  const errors: string[] = [];
+  const responseAllowedtypes = ['boolean', 'number', 'string'];
+
+  // Check fields are undefined
```

Comment:
tiny nitpick: this can be changed to "check if fields are undefined"

Reply:
I will update it!